packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -1011,25 +1025,29 @@ describe('I18n', () => {
 
   describe('#weekStartDay()', () => {
     it('uses the defaultCountry to get the week start day', () => {
-      const i18n = new I18n(defaultTranslations, {locale: 'en', country: 'FR'});
+      const i18n = new I18n(defaultTranslations, {
+        locale: 'en', country: 'FR'});
```

Comment:
?

Reply:
I ran `yarn eslint` with the `--fix` option, I believe this is the result of that? I'll revert the formatting updates.